src/controller/python/chip/ChipDeviceCtrl.py

```diff
@@ -1556,10 +1559,11 @@ def CommissionOnNetwork(self, nodeId: int, setupPinCode: int, filterType: Discov
                 self.devCtrl, nodeId, setupPinCode, int(filterType), str(filter).encode("utf-8") + b"\x00" if filter is not None else None)
         )
         if not self._ChipStack.commissioningCompleteEvent.isSet():
-            return False, -1
-        return self._ChipStack.commissioningEventRes == 0, self._ChipStack.commissioningEventRes
+            # Error 50 is a timeout
+            return PyChipError(50)
```

Comment:
Let's at least make a constant at the top of this module for CHIP_ERROR_TIMEOUT. In a follow-up, we should make a module with all CHIP_ERROR values.

Reply:
OK, Thanks